src/libs/Localize/index.ts

```diff
@@ -112,7 +172,13 @@ function getTranslatedPhrase<TKey extends TranslationPaths>(
 
     if (translatedPhrase) {
         if (typeof translatedPhrase === 'function') {
-            return translatedPhrase(...phraseParameters);
+            const result = translatedPhrase(phraseParameters);
+
+            if (typeof result === 'object') {
```

Comment:
here we can add a check for count param instead in getPluralTranslation, @brandonhenry what’s your thoughts on this ?

```javascript
if (translatedPhrase && 'count' in  translatedPhrase  && typeof result === 'object') {
     return getPluralTranslation(result, language, phraseKey, phraseParameters);
}
return result;
```